src/main/java/frc/team3388/robot/actions/shooter/ShootRPMShooter.java

```diff
@@ -0,0 +1,27 @@
+package frc.team3388.robot.actions.shooter;
+
+import com.flash3388.flashlib.robot.scheduling.actions.Action;
+import frc.team3388.robot.subsystems.Shooter;
+
+public class ShootRPMShooter extends Action {
```

Comment:
You can name it just `ShootAction` no need for other shooting options, therefore no need to specify